src/main/groovy/io/gatling/gradle/RecoverEnterprisePluginException.groovy

```diff
@@ -0,0 +1,61 @@
+package io.gatling.gradle
+
+
+import io.gatling.plugin.exceptions.*
+import org.gradle.api.BuildCancelledException
+import org.gradle.api.Task
+import org.gradle.api.logging.Logger
+import org.gradle.api.tasks.TaskExecutionException
+
+class RecoverEnterprisePluginException {
+
+    /**
+     * @param f Closure defined inside a Task
+     * @param logger used to report EnterprisePluginException
+     * @return closure result
+     */
+    static <R> R handle(Closure<R> f, Logger logger) {
```

Comment:
Groovy convention:
When a method has multiple arguments and one and only one is a Closure, put it last.

```suggestion
    static <R> R handle(Logger logger, Closure<R> f) {
```

That allows a syntax similar to currying:

```groovy
  RecoverEnterprisePluginException.handle(logger) {
    // your closure content here
  }
```